.github/workflows/pull_request.yml

```diff
@@ -1,6 +1,13 @@
 name: Build Chain
 
-on: [pull_request]
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - 'LICENSE*'
+      - '.gitignore'
+      - '*.md'
+      - '*.txt'
```

Comment:
this could be dangerous as wildcard /for changing a resource or resources/ for tests or java class?
but i can live with that danger :)

Reply:
@Ginxo Adding to what @mareknovotny said, is `*.txt` matching that file extension in the root of the repository? Or it would look into subfolders too?

Reply:
@almope @mareknovotny according to https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet `does not match slash (/)` so only `txt` files from root folder should be considered.